app/controllers/changelog_admin/entries_controller.rb

```diff
@@ -32,13 +34,41 @@ def publish
       redirect_to changelog_admin_entry_path(@entry)
     end
 
+    def edit
+      @form = ChangelogEntryForm.from_entry(@entry)
+    end
+
+    def update
+      @form = ChangelogEntryForm.from_entry(@entry)
+
+      @form.assign_attributes(form_params)
```

Comment:
Is there a reason not to use permit from Strong Paramaters here to stop a change to current_user, rather than having all the explict checks for it in the form model? Especially as we're guarding against malicious intent here. I feel like we risk rewriting stuff that we already get for free in Rails. Although I might just be misunderstanding.

Reply:
Thinking about it, I agree that we should just use strong parameters. I imagine that this model won't be used outside the request model and it would be less complicated if we use strong parameters. Thanks :)

Reply:
Awesome :)